word_break.py

```diff
@@ -0,0 +1,55 @@
+'''
+______________Asumptions__________________
+
+1. Function returns false if subset of woard is not in DIctonary
```

Comment:
Thanks for uploading your code/test cases! Don't forget to upload your pen and paper TEBOW IT for future assignments (or include pictures of your TEBOW IT on a whiteboard)! Writing the problem out by hand and timing yourself to 45 minutes is one of the best ways to practice for a real interview.